src/main/java/mate/academy/spring/vallidation/EmailValidator.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.spring.vallidation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
+    private static final String EMAIL_PATTERN = "^[\\w-.]+@([\\w-]+\\.)+[\\w-]{2,4}$";
+
+    @Override
+    public boolean isValid(String email, ConstraintValidatorContext constraintValidatorContext) {
+        return email != null && email.matches(EMAIL_PATTERN);
+    }
```

Comment:
It is better to have the constant of a type Pattern, because otherwise a new object will be created each time you call method `matches`